### PR TITLE
Release 4.0.1 version bump

### PR DESCRIPTION
**Pull request by fjnoyp**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_analytics_pinpoint/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-## 0.4.0 (2022-02-17)
+## 0.4.1 (2022-02-28)
+
+## 0.4.1 (2022-02-17)
```
Comment (ragingsquirrel3) [inline packages/amplify_analytics_pinpoint/CHANGELOG.md:3]: please restore 0.4.0 entry, same for rest

File: packages/amplify_api/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-## 0.4.0 (2022-02-17)
+## 0.4.1 (2022-02-28)
+
+- chore: clean up some code to limit analysis infos
```
Comment (ragingsquirrel3) [inline packages/amplify_api/CHANGELOG.md:3]: nit: would it be possible to keep the PR #s like past entries?

**Review by ragingsquirrel3 — APPROVED** [review]
approved w a few nits

**Review by Jordan-Nelson — COMMENTED** [review]
Other than the change to the previous change log version, this looks good. The previous change log entries should be changed back before this is merged.

File: packages/amplify_api/CHANGELOG.md (hunk 2 above)
Comment (fjnoyp) [reply packages/amplify_api/CHANGELOG.md]: Sounds good added those in

**Review by Jordan-Nelson — APPROVED** [review]

**Pull request merged**